ansible/roles/software/datagov-deploy-fgdc2iso/templates/fgdc2iso.nginx.conf.j2

```diff
@@ -0,0 +1,19 @@
+server {
+    listen 80 default_server;
+
+    server_name _;
+
+    return 301 https://$host$request_uri;
+}
+
+server {
+  listen 443 default_server ssl;
+  server_name _;
+
+  ssl_certificate {{ fgdc2iso_tls_certificate_filepath }};
+  ssl_certificate_key {{ fgdc2iso_tls_key_filepath }};
+
+  location / {
+    proxy_pass http://127.0.0.1:8080/;
```

Comment:
I'm not sure why, but I've always used [upstream](http://nginx.org/en/docs/http/ngx_http_upstream_module.html#upstream) in combination with proxy_pass. Not sure if it pools connections with proxy_pass or if there are other benefits to using upstream :shrug: